docs/server-instructions.md

```diff
@@ -111,3 +112,16 @@ Add the path to the JVM agent to the `jvm.options` file:
    -javaagent:/path/to/splunk-otel-javaagent.jar
    ```
 - Save the file and restart the server.
+
+## WebSphere Traditional
```

Comment:
Is this the commercial name, @laurit ?

Reply:
At least docker images use this name. Wikipedia page also mentions "traditional" https://en.wikipedia.org/wiki/IBM_WebSphere_Application_Server